### ISSUE TITLE
Does the MJML plug-in have an api for converting html to mjml?

### ISSUE THREAD

**Issue by yongma16** [GrapesJS/mjml#334]
The purpose is to import html and edit it in mjml. Where can I find relevant information?

**Comment by SebastianStehle** [comment]
This is not possible. MJML prints out very specific html, that would be really hard to parse. It is not that every mjml element is just one html element.

**Comment by artf** [comment]
Exactly what @SebastianStehle said, but if one day MJML devs will be able to solve that, it would be just a matter of using their API.
